<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TARUNBANSALB077" name="Regression Test Cases" tests="9" failures="4" timestamp="21 Aug 2017 06:49:30 GMT" time="519.595" errors="0">
  <testcase name="InvalidUserPassword" time="31.694" classname="TestPlans.Login">
    <failure type="java.lang.AssertionError" message="This test checks if invalid username or password throws validation. Now in this, it managed to login successfully so Incorrect Login test is not valid (Negative Scenario) expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: This test checks if invalid username or password throws validation. Now in this, it managed to login successfully so Incorrect Login test is not valid (Negative Scenario) expected [true] but found [false]
	at TestPlans.Login.InvalidUserPassword(Login.java:85)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- InvalidUserPassword -->
  <testcase name="InvalidUserPassword" time="28.422" classname="TestPlans.Login">
    <failure type="java.lang.AssertionError" message="This test checks if invalid username or password throws validation. Now in this, it managed to login successfully so Incorrect Login test is not valid (Negative Scenario) expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: This test checks if invalid username or password throws validation. Now in this, it managed to login successfully so Incorrect Login test is not valid (Negative Scenario) expected [true] but found [false]
	at TestPlans.Login.InvalidUserPassword(Login.java:85)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- InvalidUserPassword -->
  <testcase name="InvalidUserPassword" time="30.001" classname="TestPlans.Login"/>
  <testcase name="LoginLogoutCafeTown" time="65.881" classname="TestPlans.Login"/>
  <testcase name="LoginLogoutCafeTown" time="58.978" classname="TestPlans.Login"/>
  <testcase name="LoginLogoutCafeTown" time="28.923" classname="TestPlans.Login">
    <failure type="java.lang.AssertionError" message="Unable to Login into Cafe Townsend site....Login Failed expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Unable to Login into Cafe Townsend site....Login Failed expected [false] but found [true]
	at TestPlans.Login.LoginLogoutCafeTown(Login.java:57)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- LoginLogoutCafeTown -->
  <testcase name="CreateNewUser" time="136.493" classname="TestPlans.ManageUser"/>
  <testcase name="CreateNewUser" time="91.361" classname="TestPlans.ManageUser"/>
  <testcase name="CreateNewUser" time="29.257" classname="TestPlans.ManageUser">
    <failure type="java.lang.AssertionError" message="Unable to Login into Cafe Townsend site....Login Failed expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Unable to Login into Cafe Townsend site....Login Failed expected [false] but found [true]
	at TestPlans.ManageUser.CreateNewUser(ManageUser.java:60)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- CreateNewUser -->
</testsuite> <!-- Regression Test Cases -->
